backend/services/src/shared/casl/policy.guard.ts

```diff
@@ -141,9 +154,14 @@ export const PoliciesGuardEx = (injectQuery: boolean, action?: Action, subject?:
         }
       }
 
-      return policyHandlers.every((handler) =>
-        this.execPolicyHandler(handler, ability, body),
+      const pHandler = policyHandlers.every((handler) =>
+        this.execPolicyHandler(handler, ability, body)
       );
+      if (pHandler) {
+        return pHandler;
+      } else {
+        throw new ForbiddenException(unAuthErrorMessage);
+      }
```

Comment:
if policyHandlers.length == 0 then it wont reach here. So I think we need handle this inside the policyHandlers.length == 0 block also. please check

Reply:
okay